src/common-model/functions.ts

```diff
@@ -70,7 +70,10 @@ export function getClinicalObjectsFromDonor(
     }
   }
 
-  if (clinicalEntitySchemaName === ClinicalEntitySchemaNames.TREATMENT) {
+  if (
+    clinicalEntitySchemaName === ClinicalEntitySchemaNames.TREATMENT ||
+    clinicalEntitySchemaName === ClinicalEntitySchemaNames.SURGERY
+  ) {
```

Comment:
these nested `ifs`.... not sure if there's a reason for them but there's a lot here already

Reply:
can put entire condition in a single if

Reply:
At first I thought this was just about combining the two if statements, like follows
```TypeScript
[ClinicalEntitySchemaNames.TREATMENT, ClinicalEntitySchemaNames.SURGERY].includes(clinicalEntitySchemaName) && donor.treatments```

But then i looked at the rest of the function... definitely fine to continue this pattern as is.

This probably should've been written as a switch:
```TypeScript
switch(clinicalEntitySchemaName( {
  case ClinicalEntitySchemaNames.FAMILY_HISTORY:
   return donor.familyHistory || [];
  case ClinicalEntitySchemaNames.TREATMENT:
    // fall through to SURGERY
  case ClinicalEntitySchemaNames.SURGERY:
   return donor.treaments || [];
   // ... etc for each SchemaName
  default:
    return [];
```
But even that is messy because of the alternate return type.

Reply:
Actual review comment:
In the case of `clinicalEntitySchemaName === SURGERY` I think we only want to return the surgery treatments. The code on line 106 handles that case. Why is surgery so special that it is instead included here?

Reply:
@joneubank that is a very valid question however there is definitely a bug here, although I think the local Mongo data I'm using may be the issue

I can find several examples of `donor.treatments > treatment.clinicalInfo.therapy_type === 'Surgery'` but cannot find any data (using local Mongodb collection) where `donor.treatments > treatment.therapies[n].therapyType === 'surgery'`

Therapies makes the most sense and I did notice a mismatch btwn data in Clinical Info and the surgery schema fields which suggested something was wrong

But w/ no data representing a case where `therapies[n].therapyType === 'surgery'` I totally overlooked the block on line 106. I had to return `donor.treatments` to even _find_ 'Surgery' data

Reply:
@joneubank on further inspection there's actually a working example of this page in Dev > DATA-CA, I think the issue needs to be better defined to specific env/program where the bug is occurring to investigate further